checkbox-ng/checkbox_ng/urwid_ui.py

```diff
@@ -691,11 +691,11 @@ def interrupt_dialog(host):
         ('start', 'dark green,bold', 'black'),
     ]
     choices = [
-        _("Cancel the interruption and resume the session (ESC)"),
-        _("Kill test in progress and move on to next"),
-        _("Disconnect the master (Same as CTRL+C)"),
-        _("Stop the checkbox slave @{}".format(host)),
-        _("Abandon the session on the slave @{}".format(host)),
+        _("Nothing, continue testing (ESC)"),
+        _("Stop the test in progress and move on to the next"),
```

Comment:
If I select this option, what will happen to that test? Will it be marked as `skipped` or `fail`? Or something else? I actually tried and... crashed Checkbox :D → #550

Reply:
This should be fixed by the drive-by fix that I have done in this PR! That `if ... is None: ...`

Reply:
Awesome! In that case, you should specify this in the commit message. I was not sure why you had changed this part of the code to be honest. Think of commit messages as a help for future archeologists: you in 3 months, me and everyone in the team ;)

Reply:
For reference: https://github.com/canonical/checkbox/pull/540/commits/1f5815c7a41e60527f3848b72983c771cc63272c